site/management.md

```diff
@@ -288,25 +288,31 @@ rabbitmqctl set_user_tags full_access administrator
 RabbitMQ can be configured to use [JWT-encoded OAuth 2.0 access tokens](https://github.com/rabbitmq/rabbitmq-auth-backend-oauth2) to authenticate client applications and management UI users. When doing so, the management UI does
 not automatically redirect users to authenticate
 against the OAuth 2 server, this must be configured separately. Currently,
-RabbitMQ has been tested against the following Authorization servers:
+**Authorization code flow with PKCE** has been tested against the following Authorization servers:
```

Comment:
Use present tense where possible in documentation: **Authorization code flow with PKCE** is tested with the following Authorization servers: